src/app/pages/test-report/cat-cpc/components/cpc-end-test-modal/components/modal-result-item/modal-result-item.ts

```diff
@@ -33,6 +36,9 @@ export class ModalResultItemComponent {
 
   displayScore(score: number): string {
     const finalScore = (score || 0);
+    if (this.testState > 0) {
+      return this.adi3 ? (score || 0).toString() : `${finalScore}%`;
+    }
     return this.adi3 ? (score || '-').toString() : `${finalScore}%`;
   }
```

Comment:
```
  displayScore(score: number): string {
    const finalScore = (score || 0);
    return this.adi3 ? (typeof score === 'number' ? score || '-').toString() : `${finalScore}%`;
  }
```
Think we can change to the above and then not need to pass in a new input to component